cluster/vagrant/provision-master.sh

```diff
@@ -54,6 +54,7 @@ for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
     echo "Adding $minion to hosts file"
     echo "$ip $minion" >> /etc/hosts
   fi
+  echo "127.0.0.1 localhost" >> /etc/hosts
```

Comment:
oh, never mind, I thought you were  saying "why did you put this in provision-minion.sh"... 

the answer is *no* I don't need this in minion, the expectation of the test is that the master is able to run "get pods" against itself.  Wether or not the minions are running this command is not needed....

makes sense?